apps/dotcom/src/components/StoreErrorScreen.tsx

```diff
@@ -26,6 +26,11 @@ export function StoreErrorScreen({ error }: { error: Error }) {
 					'Your changes were rejected by the server. Please reload the page. If the problem persists contact the system administrator.'
 				break
 			}
+			case TLIncompatibilityReason.RoomNotFound: {
+				message =
+					'The room you are trying to connect to does not exist. Please check the URL and try again. If the problem persists contact the system administrator.'
```

Comment:
Let's have this message just read "The room you are trying to connect to does not exist.":

- "Please check the URL and try again." probably isn't actionable for most users - the URL is probably a random ID
- "If the problem persists contact the system administrator." is definitely not actionable imo

I'd also maybe suggest adding a "Back to tldraw" link. Maybe we can say if we're in an iframe, we make it read "Open tldraw" with `target="_blank"`

Reply:
good call, i ended up consolidating this with ErrorPage so that they have some consistency between them.

<img width="1344" alt="Screenshot 2024-04-23 at 14 13 17" src="https://github.com/tldraw/tldraw/assets/469604/27357318-7442-416c-8bc7-ad7222ea6d1e">